static/js/app.js

```diff
@@ -87,6 +87,14 @@ function untoast() {
   clearTimeout(toastTimeout);
 }
 
+function rename(name) {
+  if (!config.name) {
+    config.name = "";
+  }
```

Comment:
It won't do that. `!1` is `false`, so it won't trigger this. And anyways you overwrite `config.name` in the next breath, so any decent compiler would optimize this if condition away.